install.vbs

```diff
@@ -72,7 +72,7 @@ Call CopySymlinkFiles()
 
 ' Getting npm dependencies (for react-native)
 WScript.Echo "Getting npm dependencies"
-intReturnVal = objShell.Run("npm install react-native@0.35.0 react@15.3.1 --silent", 1, True)

```

Comment:
Looking at https://github.com/facebook/react-native/blob/v0.43.0/package.json#L129,
I think it should be react@16.0.0-alpha.6